src/pages/BookDetails/BookDetails.module.scss

```diff
@@ -0,0 +1,7 @@
+.gridContainer {
+  background-color: lightgrey;
+  padding-bottom: 40px;
+  padding-top:40px;
```

Comment:
this is probably because the page is a bit below the header right? Maybe we can add a `<Box>` around `<Outlet>` in `DrawerAppBar.js` with some global (responsive) margins and padding, that way you don't have to add it in every page